tree/treeplayer/inc/TTreeReaderValue.h

```diff
@@ -132,14 +132,22 @@ class TTreeReaderValue: public ROOT::Internal::TTreeReaderValueBase {
       TTreeReaderValueBase(&tr, branchname,
                            TDictionary::GetDictionary(typeid(NonConstT_t))) {}
 
+   /// Return a pointer to the value of the current entry.
+   /// Return a nullptr and print an error if no entry has been loaded yet.
+   /// The returned address is guaranteed to stay constant while a given TTree is being read from a given file.
+   /// It might change when the underlying TTree/TFile is switched, e.g. when a TChain switches files.
    T* Get() {
       if (!fProxy){
          Error("TTreeReaderValue::Get()", "Value reader not properly initialized, did you remember to call TTreeReader.Set(Next)Entry()?");
-         return 0;
+         return nullptr;
       }
       void *address = GetAddress(); // Needed to figure out if it's a pointer
       return fProxy->IsaPointer() ? *(T**)address : (T*)address; }
+   /// Return a pointer to the value of the current entry.
+   /// Equivalent to Get.
```

Comment:
Get -> `Get()`?

Reply:
I hoped Doxygen was smart enough to highlight that and add an anchor link to the `Get` method I mention. Not sure if that's the case, I will test it locally.